centaur/src/main/scala/centaur/test/Test.scala

```diff
@@ -386,7 +398,6 @@ object Operations {
         validateMetadata(workflow, expectedMetadata).handleErrorWith({ _ =>
           for {
             _ <- IO.sleep(2.seconds)
-            _ = if (LogFailures) Console.err.println(s"Metadata mismatch for ${submittedWorkflow.id} - retrying")
```

Comment:
this does get printed if it never reaches the expected value?

Reply:
Correct - the actual message (including *which* metadata was wrong) gets printed if this "eventuallyMetadata" times out